moto/ec2/responses/elastic_network_interfaces.py

```diff
@@ -1,3 +1,4 @@
+from moto.awslambda.exceptions import InvalidParameterValueException
```

Comment:
This should come from EC2's exceptions (`from .exceptions import ..`)

I don't know if there is an exception already, but if not, we can just recreate it there